DAL/DashboardDAO.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Model.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MongoDB.Driver;
+using MongoDB.Bson;
+using Model;
+
+namespace DAL
+{
+    public class DashboardDAO : DAO
+    {
+        public int CountAllOpenTickets()
+        {
+            List<Ticket> tickets = GetTicketCollection().Find(ticket => ticket.Status == ETicketStatus.Open).ToList();
+            return tickets.Count;
+        }
+        public int CountAllTickets()
+        {
+            List<Ticket> tickets = GetTicketCollection().Find(ticket => ticket.Status != null).ToList();
+            return tickets.Count;
```

Comment:
Dit kan in een methode in de service. 
List<Ticket> tickets = GetTicketsFromDatabase();
int openTicketCount = tickets.Count(ticket => ticket.Status == "open");
int ticketCount = tickets.Count();